src/server/services/tips/predicates.ts

```diff
@@ -137,6 +137,23 @@ export const hasKidsBetweenAges4And11: TipsPredicateFN = (
   return hasKidsBetweenAges(appState.BRP?.content?.kinderen, 4, 11, today);
 };
 
+export const hasOldestKidBornFrom2016: TipsPredicateFN = (appState) => {
+  console.log(
+    '=>(predicates.ts:142) appState.BRP?.content',
+    appState.BRP?.content
+  );
+  const oldestKid = appState.BRP?.content?.kinderen?.sort(
```

Comment:
Je zou ook src/universal/helpers/`sortByNumber` kunnen gebruiken.

Reply:
Om sortbynumber te gebruiken moet ik eerst de geboortedatum omzetten naar epochtime in een map, dan kan ik sorteren. Daarna zou ik de ms weer om moeten zetten naar de geboorte datum oid om 
  const birthYear = new Date(oldestKid?.geboortedatum).getFullYear();
  return birthYear >= 2016 && birthYear < 2024 te doen. Geloof dat de huidige manier korter is. Maar kan er naast zitten.